app/utils/formValidation.ts

```diff
@@ -0,0 +1,22 @@
+export const validateMinLength = (minLength: number, field: string) => (value: string) => {
+  return value.trim().length < minLength ? `${field} must have at least ${minLength} letters. Please try again.` : null;
+}
+
+/**
+ * Validates the format of an email address.
+ * @param {string} value - The email address to validate.
+ * @returns {string|null} An error message if the email format is invalid (local-part@domain.TLD, with optional subdomains), otherwise null.
+ **/
+export const validateEmail = (value: string) => {
+  if (value.trim().length < 3) {
+    return 'Email must have at least 3 letters. Pleaes try again.';
```

Comment:
Just caught the typo here.

Reply:
Oh and is there a reason you aren't using your validateMinLength function?

Reply:
Refactored the helpers to make it more DRY.

Reply:
Where is the typo btw?

Reply:
Tks! The code has been updated with the correct spelling:)